src/crypto/litecoin.ts

```diff
@@ -85,7 +87,16 @@ class Litecoin implements ICrypto {
 
   // eslint-disable-next-line class-methods-use-this
   validateAddress(address: string): boolean {
-    return fallbackValidateAddress(address);
+    // bech32 address
+    if (address.substring(0, 3) === 'ltc') {
+      if (bech32.decodeUnsafe(address) || bech32m.decodeUnsafe(address)) {
```

Comment:
Just making sure: Do litecoin actually use bech32m?

Reply:
good point. I've removed bechm for litecoin and digibyte